workflows/kneaddata_paired.smk

```diff
@@ -18,7 +18,7 @@ rule kneaddata:
         rev = temp(os.path.join(kneadfolder, "{sample}_kneaddata_paired_2.fastq")),
         log = os.path.join(kneadfolder, "{sample}_kneaddata.log")
     run:
-        shell("kneaddata --input {{input.fwd}} --input {{input.rev}} --reference-db /hg37 --output {} --output-prefix {{wildcards.sample}}_kneaddata --trimmomatic /opt/conda/share/trimmomatic".format(kneadfolder))
+        shell("kneaddata qsub -t {cluster.processors} --input {{input.fwd}} --input {{input.rev}} --reference-db /hg37 --output {} --output-prefix {{wildcards.sample}}_kneaddata --trimmomatic /opt/conda/share/trimmomatic".format(kneadfolder))
```

Comment:
The trouble with this approach is that, if we're not running on a cluster, we don't have `qsub`. Also, our cluster uses `SLURM` where the equivalent command is `srun` :-)

It might require some kind of conditional statement somewhere. Like "If we find `cluster.processors`, use that value, otherwise use `1`...